Content.Server/GameObjects/EntitySystems/BuckleSystem.cs

```diff
@@ -26,6 +26,14 @@ public override void Initialize()
             SubscribeLocalEvent<EntRemovedFromContainerMessage>(ContainerModified);
         }
 
+        public override void Update(float frameTime)
+        {
+            foreach (var comp in ComponentManager.EntityQuery<BuckleComponent>())
```

Comment:
ignore paused.

Reply:
You mean like this?
```
if(!comp.Paused)
    comp.Update();
```

Reply:
you can pass in false to the EntityQuery for it.

Reply:
Fixed